package.json

```diff
@@ -12,6 +12,7 @@
     "build": "npm run build:browser && npm run build:main",
     "build:browser": "tsc --project tsconfig.json --module ESNext --outDir ./browser",
     "build:main": "tsc --project tsconfig.json --module CommonJS --outDir ./build",
+    "prepare": "npm run build:main",
```

Comment:
```suggestion
```

Reply:
Reverted, but this _is_ handy for anyone who needs to install a fork of the project. Could conceivably drop this `build` in favor of "prepare": "npm run build":
https://github.com/sergiodxa/remix-utils/blob/68f7d5d2faf3bb429c78d9f48e26fa6834961048/.github/workflows/publish.yml#L21-L23

Since [`prepare` runs before `publish`](https://docs.npmjs.com/cli/v6/using-npm/scripts#life-cycle-scripts).